### PR TITLE
Initial commit of Datums

### PR DESCRIPTION
**Pull request by joshmarlow**
@micahhausler please review.  Let me know if I missed anything for 1.7 support.

### CODE REVIEW

**Comment by micahhausler** [comment]
Could you run through this checklist for Django 1.7 apps? https://github.com/micahhausler/django-app-upgrade-guide

File: setup.py [hunk 1]
```diff
@@ -39,16 +39,21 @@ def get_version():
     ],
     license='MIT',
     install_requires=[
-        'django>=1.6',
+        'coverage>=3.7.1',
+        'Django>=1.6.5,<1.8',
```
Comment (micahhausler) [inline setup.py]: Could you just make this `Django>=1.7`?

**Comment by micahhausler** [comment]
You'll need to update `.travis.yml` to pin to `DJANGO=1.7.7`.
If you could also drop Python 3.3 in `.travis.yml` and your setup.py classifiers

and also change the doc build line in `.travis.yml` from `cd docs && make html` to `python setup.py build_sphinx`

**Comment by micahhausler** [comment]
I've not gotten to update the django template yet, but could you add the following to your `setup.cfg`?
https://github.com/ambitioninc/django-manager-utils/blob/develop/setup.cfg#L7-L13

File: setup.py [hunk 2]
```diff
@@ -39,16 +39,21 @@ def get_version():
     ],
     license='MIT',
     install_requires=[
-        'django>=1.6',
+        'coverage>=3.7.1',
```
Comment (micahhausler) [inline setup.py]: Could you move coverage to be a test requirement?

File: setup.py [hunk 3]
```diff
@@ -39,16 +39,21 @@ def get_version():
...
+        'Django>=1.6.5,<1.8',
+        'jsonfield',
+        'six',
+        'psycopg2',
```
Comment (micahhausler) [inline setup.py]: Could you move this to be a test requirement? Although most users will probably use psycopg2, they may be on MySQL

**Comment by joshmarlow** [comment]
@micahhausler thanks for the comments.  They are addressed.

File: setup.py [hunk 4]
```diff
@@ -39,16 +38,21 @@ def get_version():
     ],
     license='MIT',
     install_requires=[
-        'django>=1.6',
+        'Django>=1.7',
+        'jsonfield',
+        'six',
```
Comment (micahhausler) [inline setup.py]: You can remove six, as it is vendored inside of Django https://github.com/django/django/blob/master/django/utils/six.py

**Comment by micahhausler** [comment]
LGTM! :+1:

**Pull request merged**